cli/flox-rust-sdk/src/utils/gomap.rs

```diff
@@ -0,0 +1,37 @@
+use std::collections::BTreeMap;
+
+use serde::{Deserialize, Serialize};
+
+/// Representation of Go's `struct{}`.
+#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq)]
+pub struct GoEmptyStruct(BTreeMap<(), ()>);
+
+/// Representation of Go's `map[string]struct{}`.
+#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq)]
+pub struct GoMap(BTreeMap<String, GoEmptyStruct>);
+
+impl From<Vec<String>> for GoMap {
+    fn from(v: Vec<String>) -> Self {
+        GoMap(
+            v.into_iter()
+                .map(|s| (s, GoEmptyStruct::default()))
+                .collect(),
+        )
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use serde_json;
+
+    use super::*;
+
+    #[test]
+    fn new() {
```

Comment:
This test could use a slightly more descriptive name

Reply:
Done at the same time as moving to sets.